distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -66,13 +69,25 @@ type Point_Data
     ## PRIVATE
     fallback_column : Table -> Column ! No_Fallback_Column
     fallback_column self table = case self of
-        Point_Data.X -> Point_Data.iota table.row_count
+        Point_Data.X ->
+            candidate = table.columns.first
+
+            if candidate.is_numeric then candidate else Point_Data.iota table.row_count
         Point_Data.Y ->
             x_column = Point_Data.X.lookup_in table
             candidates = table.columns
             is_good_enough c = c.is_numeric && c.name != x_column.name
             is_good c = is_good_enough c && (self.is_recognized c).not
 
+            candidates.find if_missing=(Error.throw Nothing) is_good . catch_ <| candidates.find is_good_enough
+        Point_Data.Y_multi _ ->
+            x_column = Point_Data.X.lookup_in table
+            y_column = Point_Data.Y.lookup_in table
+            candidates_filtered = table.columns.filter (c-> c.is_numeric && c.name != x_column.name && c.name != y_column.name)
+            candidates = candidates_filtered.slice self.number (candidates_filtered.length)
```

Comment:
If I understand your intent correctly, the same thing is achieved by:
```suggestion
            candidates = candidates_filtered.drop self.number
```

I prefer `take`/`drop` to `slice` where applicable, because it is more immediately clear what you want to do - drop the first `self.number` entries from the vector.